balboa-http/src/main/scala/com/socrata/balboa/server/MainServlet.scala

```diff
@@ -0,0 +1,67 @@
+package com.socrata.balboa.server
+
+import javax.servlet.http.HttpServletRequest
+
+import com.socrata.balboa.BuildInfo
+import com.socrata.balboa.metrics.data.BalboaFastFailCheck
+import com.socrata.balboa.server.rest.{EntitiesRest, MetricsRest}
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import org.json4s.{DefaultFormats, Formats}
+import org.scalatra.{InternalServerError, NotFound, Ok, ScalatraServlet}
+import org.scalatra.json.JacksonJsonSupport
+
+import scala.collection.JavaConverters._
+
+class MainServlet extends ScalatraServlet with StrictLogging with JacksonJsonSupport {
+
+  protected implicit val jsonFormats: Formats = DefaultFormats
+
+  val versionString = BuildInfo.toJson
+
+  def getAccepts(req: HttpServletRequest): Seq[String] = {
+    req.getHeaders("accept").asScala.toSeq
+  }
+
+  get("/*") {
+    contentType = "application/json"
+    NotFound(Error(404, "Not found."))
```

Comment:
NotFound is a 404. Is it necessary to specify anything? What about:

    NotFound()

or:

    NotFound(reason = "Not found.")